ArjunaJTA/jta/classes/com/arjuna/ats/internal/jta/recovery/arjunacore/XARecoveryModule.java

```diff
@@ -1242,4 +1254,4 @@ private enum ScanStates {
     private Set<String> contactedJndiNames = new HashSet<String>();
 
 	private static XARecoveryModule registeredXARecoveryModule;
-}
\ No newline at end of file
+}
```

Comment:
A little bit of formatting I would ask to be removed, please? I hope that is OK with you